docs/diagrams/CountryNotePanelClientViewActDiag.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+start
+:User inputs a command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+:LogicManager executes the user command
+and returns an instance of CommandResult;
+
+if () then ([commandResult.shouldDisplayClient()])
+    :Remove all child components in
+    the Display Panel Container;
+    :Add the Container of the
+    Client View UI as a child component
+    of the Display Panel Container;
+elseif () then ([commandResult.shouldDisplayCountryNote()])
+    :Remove all child components in
+    the Display Panel Container;
+    :Add the Container of the
+    Country Note Panel UI as a child component
+    of the Display Panel Container;
+else ([else])
+
```

Comment:
I think using a merge node to merge all arrows before pointing to end will be good, not sure if it's a limitation of PlantUML that there are double arrows.

Reply:
Similar to comment above